.github/workflows/docs.yaml

```diff
@@ -0,0 +1,40 @@
+name: Docs
+
+"on":
+  push:
+    branches:
+      - main
+      - master
+      - "tickets/**"
+      - "u/**"
+    paths:
+      - "docs/**"
+      - "src/**.py"
```

Comment:
I'd also add CHANGELOG.rst in general because we often symlink the change log into the docs.

Now one thing I'm unsure about is whether the `paths` field would prevent docs from being built for tags. My suspicion is that it won't. In general, it'd be good to have a documentation edition build for each tag, so we might want to double-check how this pans out before applying this to other repos.

Reply:
I think you may be right that the tags are affected as well, but after adding `CHANGELOG.rst`, at least one affected file should be changed with each new tag.